### ISSUE TITLE
Installation of Elastic Search Version 6.6.0 Failed

### ISSUE THREAD

**Issue by upchair** [elastic/windows-installers#255]
### Steps to reproduce the behaviour

1.
2.
3.

**Comment by codebrain** [comment]
Hi @upchair - are you able to provide any more details? Perhaps the installation log?

**Comment by upchair** [comment]
Dear Codebrain,

Thanks for your response. Please find attached the the installation log file AND the elasticsearch log file as you requested 

Thank You. 

[elasticsearch.log](https://github.com/elastic/windows-installers/files/2863640/elasticsearch.log)
[MSI46364.LOG](https://github.com/elastic/windows-installers/files/2863643/MSI46364.LOG)

**Comment by Mpdreamz** [comment]
It seems `6.5.0` was already running on the machine but was not installed through the MSI. Is this correct @upchair ?

**Comment by upchair** [comment]
@codebrain AND @Mpdreamz 

I have resolved the problem. The problem was with the java update that was carried out a day earlier on my system/laptop. Apparently, the JRE folder where the update was carried out did not include the java.exe file. That folder was empty which I found rather strange because java updates usually include this file in the JRE folder.

When I noticed this I had to go to the oracle website myself and manually install the proper/appropriate latest LTS based Java update and I made sure that the java.exe file was included in the appropriate folder this time and boom the elastic search server installed successfully . 

Thanks guys for your time and commitment. I really appreciate it.

Richard

**Comment by Mpdreamz** [comment]
Really appreciate you taking the time to give us this update @upchair. 

I have not seen this happening before but now we now this is something to look out for in the future :+1:

**Comment by upchair** [comment]
@Mpdreamz thanks for the compliment. I just felt that it is proper for me to update for 2 reasons: (1) To inform those who contributed to helping me resolve the problem ( Mpdream & Codebrain) about how I resolved the problem. (2) To inform and share with those who may encounter a similar problem in the future whose solution to the problem may just be exactly like the one I narrated above.

For me there is nothing as rewarding as coming to forums like github and succeed in finding answers to problems one encounters 😀
